bin/src/upload.dart

```diff
@@ -27,7 +27,7 @@ class UploadPart extends CommandLinePart {
       });
     }
 
-    var pRoot = new Directory(res["package-root"]);
+    var pRoot = new File(res["packages"]);
```

Comment:
Can we provide some way that is also backward compatible? As of now, every script that used `package-root` would break. I'd rather still support it but provide deprecation warnings.